SquareLine/boards/custom_waveshare_7inch/components/ws_7inch/README.md

```diff
@@ -0,0 +1,53 @@
+# BSP: Custom WaveShare 7inch LCD 
+
+* [Hardware Reference](https://www.waveshare.com/wiki/7inch_Capacitive_Touch_LCD_(C))
+
+This is example how to make custom BSP and generate custom board for SquareLine.
+
+![image](pic.jpg)
```

Comment:
image.png and include/bsp/pic.jpg seem like the same image. Do we need two copies?

Reply:
Yes, there must be two copies, because the generator take one into SquareLine package with this name and second one is used in component readme - this cannot links to SquareLine image.